addon/components/ui-message.js

```diff
@@ -262,16 +252,16 @@ export default Ember.Component.extend({
   },
 
   /**
-    Send 'onShow' action with component itself as an argument.
-
-    @method _show
-    @private
-  */
-  show() {
-    this.set('visible', true);
-
-    this.sendAction('onShow', this);
-  },
+  Processing 'visible' flags status.
+   */
```

Comment:
Comment to this method doesnt satisfy our YUIDoc requirements. Must be something like that:
`Observes 'visible' property.`
`Sends actions 'onShow' & 'onHide'.`
``
`@method _didVisibilityChange`
`@private`